src/_base/harness/attributes/environment/local.yml

```diff
@@ -3,4 +3,7 @@ attributes:
   app:
     build: dynamic
     mode: development
-    version: develop
\ No newline at end of file
+    version: develop
+  console:
+    healthcheck:
+      enabled: true
```

Comment:
can it be enabled for helm and CI as well? It's useful for seeing the deployment readiness, and CI should act the same as local

Reply:
technically helm way is enough being a common `console.healthcheck.enabled`, as it'll get the same file CI was built from

Reply:
note though that the helm way is hard coded to `app state`